hub/core/tests/common.py

```diff
@@ -6,8 +6,18 @@
 LOCAL = "local"
 S3 = "s3"
 
+
+def c(*args):
```

Comment:
nit-pick: can we rename this method.

Reply:
i just don't want it to be typed a lot, it would much up visibility if it is longer than a couple chars

Reply:
it's pretty apparent that it will only be used here and the method is 5 lines from where it's being used. i don't think we need to hold peoples hands with literally every function def. if it's obvious what it does, why use more characters than necessary

Reply:
I don't think you need it at all. 
Before:
```
ALL_CACHES = [c(MEMORY, LOCAL), c(MEMORY, S3), c(LOCAL, S3), c(MEMORY, LOCAL, S3)]
```
After:
```
ALL_CACHES = list(map(lambda i: ','.join(i), 
    [(MEMORY, LOCAL), (MEMORY, S3), (LOCAL, S3), (MEMORY, LOCAL, S3)]
))
```

Reply:
sure will do this

Reply:
done